src/oncall/notifier/reminder.py

```diff
@@ -48,8 +48,12 @@ def reminder(config):
     cursor.execute('SELECT `last_window_end` FROM `notifier_state`')
     if cursor.rowcount != 1:
         window_start = int(time.time() - interval)
-        logger.warning('Corrupted/missing notifier state; unable to determine last window. Guessing %s',
+        logger.warning('Corrupted/missing notifier state; unable to determine last window. Guessing %s. Creating state in database',
                        window_start)
+        # create a clean state in the datebase
+        cursor.execute('TRUNCATE `notifier_state`')    
```

Comment:
Fix Flake8 errors and make sure circleci tests are passing

```src/oncall/notifier/reminder.py:54:52: W291 trailing whitespace```

Reply:
@diegocepedaw Fixed